diff --git a/target/product/core.mk b/target/product/core.mk
index dfce36015..1e7ed122f 100644
--- a/target/product/core.mk
+++ b/target/product/core.mk
@@ -21,10 +21,7 @@
 # core_tiny.mk in addition to core_minimal.mk.
 
 PRODUCT_PACKAGES += \
-    BasicDreams \
     BlockedNumberProvider \
-    BookmarkProvider \
-    Browser2 \
     BuiltInPrintService \
     Calendar \
     CalendarProvider \
@@ -34,7 +31,6 @@ PRODUCT_PACKAGES += \
     DeskClock \
     DocumentsUI \
     DownloadProviderUi \
-    Email \
     ExactCalculator \
     ExternalStorageProvider \
     FusedLocation \
diff --git a/target/product/core_minimal.mk b/target/product/core_minimal.mk
index b8e04e0ca..96ac275fb 100644
--- a/target/product/core_minimal.mk
+++ b/target/product/core_minimal.mk
@@ -25,19 +25,15 @@ PRODUCT_NAME := core
 PRODUCT_PACKAGES += \
     BackupRestoreConfirmation \
     CompanionDeviceManager \
-    CtsShimPrebuilt \
-    CtsShimPrivPrebuilt \
     DownloadProvider \
     ExtShared \
     ExtServices \
-    HTMLViewer \
     MediaProvider \
     PackageInstaller \
     SecureElement \
     SettingsProvider \
     Shell \
     StatementService \
-    WallpaperBackup \
     android.hidl.base-V1.0-java \
     android.hidl.manager-V1.0-java \
     bcc \
diff --git a/target/product/core_tiny.mk b/target/product/core_tiny.mk
index 4ef7b48fa..3ca1f7024 100644
--- a/target/product/core_tiny.mk
+++ b/target/product/core_tiny.mk
@@ -36,15 +36,12 @@ PRODUCT_PACKAGES += \
 
 PRODUCT_PACKAGES += \
     BackupRestoreConfirmation \
-    CtsShimPrebuilt \
-    CtsShimPrivPrebuilt \
     DefaultContainerService \
     ExtShared \
     ExtServices \
     SecureElement \
     SettingsProvider \
     Shell \
-    WallpaperBackup \
     android.hidl.base-V1.0-java \
     android.hidl.manager-V1.0-java \
     bcc \
diff --git a/target/product/generic_no_telephony.mk b/target/product/generic_no_telephony.mk
index e266cc36d..f0ab77d35 100644
--- a/target/product/generic_no_telephony.mk
+++ b/target/product/generic_no_telephony.mk
@@ -20,9 +20,6 @@
 PRODUCT_PACKAGES := \
     Bluetooth \
     BluetoothMidiService \
-    Camera2 \
-    Gallery2 \
-    Music \
     MusicFX \
     OneTimeInitializer \
     Provision \
@@ -31,7 +28,6 @@ PRODUCT_PACKAGES := \
     DisplayCutoutEmulationDoubleOverlay \
     DisplayCutoutEmulationCornerOverlay \
     DisplayCutoutEmulationTallOverlay \
-    EasterEgg \
     WallpaperCropper
 
 PRODUCT_PACKAGES += \
diff --git a/target/product/sdk_base.mk b/target/product/sdk_base.mk
index 1e5ed1928..0745b828b 100644
--- a/target/product/sdk_base.mk
+++ b/target/product/sdk_base.mk
@@ -17,16 +17,11 @@
 PRODUCT_PROPERTY_OVERRIDES :=
 
 PRODUCT_PACKAGES := \
-	CellBroadcastReceiver \
 	CubeLiveWallpapers \
 	CustomLocale \
-	Development \
 	Dialer \
-	Gallery2 \
 	Launcher3QuickStep \
-	Camera2 \
 	librs_jni \
-	LiveWallpapersPicker \
 	Mms \
 	Music \
 	Protips \
@@ -37,7 +32,6 @@ PRODUCT_PACKAGES := \
 	sqlite3 \
 	SystemUI \
 	SysuiDarkThemeOverlay \
-	EasterEgg \
 	WallpaperPicker \
 	WidgetPreview \
 
