diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 8ddf7f6..cfb240d 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -83,10 +83,6 @@ TARGET_USES_C2D_COMPOSITION := true
 TARGET_USES_ION := true
 USE_OPENGL_RENDERER := true

-# FM
-BOARD_HAVE_QCOM_FM := true
-TARGET_QCOM_NO_FM_FIRMWARE := true
-
 # Filesystem
 TARGET_ANDROID_FILESYSTEM_CONFIG_H := $(VENDOR_PATH)/android_filesystem_config.h

diff --git a/msm8916.mk b/msm8916.mk
index 605511c..7afdc39 100644
--- a/msm8916.mk
+++ b/msm8916.mk
@@ -85,10 +85,6 @@ PRODUCT_PACKAGES += \
     libbson \
     Snap

-# CMActions
-PRODUCT_PACKAGES += \
-    CMActions
-
 # Display
 PRODUCT_PACKAGES += \
     gralloc.msm8916 \
@@ -108,11 +104,6 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/configs/extract_firmware.sh:install/bin/extract_firmware.sh
 endif

-# FM
-PRODUCT_PACKAGES += \
-    FMRadio \
-    libfmjni
-
 # For android_filesystem_config.h
 PRODUCT_PACKAGES += \
     fs_config_files
